CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ v1.0 (in development, master branch)
 - ENH: added bots.parsers.spamhaus.parser_cert
 - ENH: added bots.parsers.fraunhofer.parser_dga
 - ENH: added bots.experts.certat_contact.expert
+- ENH: added bots.experts.reducer.expert
```

Comment:
how about calling it "field_reducer"? Why?

Because a "reducer" sounds like the reducer part of map-reduce.
That's seomthing else.

However, TBD... not sure... anyway, not a big issue.